admin/commands/base.go

```diff
@@ -184,6 +184,31 @@ func ParseDisableCollectors(collectors []string) []string {
 	return disableCollectors
 }
 
+// ValidateEnvironmentVariableNames validates environment variable names.
+func ValidateEnvironmentVariableNames(varNames []string) ([]string, error) {
+	if len(varNames) == 0 {
+		return nil, nil
+	}
+
+	result := make([]string, 0, len(varNames))
+	validNamePattern := regexp.MustCompile(`^[A-Z_][A-Z0-9_]*$`)
+
+	for _, name := range varNames {
+		name = strings.TrimSpace(name)
+		if name == "" {
```

Comment:
If the variable name is empty, we may have to signal an error to the user, who has probably no intention to pass empty names. WDYT?